kafka/adminapi.go

```diff
@@ -1097,6 +1099,16 @@ func (a *AdminClient) cToAuthorizedOperations(
 	return authorizedOperations
 }
 
+// cToUuid converts a C rd_kafka_uuid_t to a Go Uuid.
+func (a *AdminClient) cToUuid(cUuid *C.rd_kafka_uuid_t) Uuid {
```

Comment:
if rd_kafka_uuid_t is null in the argument, then what happens here? 
we should add a null check and return the 'zero' UUID, or else make this return a *Uuid and return a nil pointer

(I think for current situation it won't ever be nil but this method will also be used for future methods, possibly, or behavior might change)

Reply:
Currently, we won't get nil. I think let's update this method later for the nil case as we are not even making any constructor for this field for now.